app/pages/game/birthle/Birthle.jsx

```diff
@@ -1,14 +1,23 @@
-import React from "react";
+import React, {Component} from "react";
 import { useEffect, useRef, useState } from "react";
 import "./Birthle.css";
 import fetchSlugs from "./fetchSlugs";
 import fetchPersons from "./fetchPersons";
 import Result from "./components/Result/Result";
 import Game from "./components/Game/Game";
 import useTrait from "./useTrait";
+import ConsentForm from "./components/ConsentForm/ConsentForm";
+import DemographicForm from "./components/DemographicForm/DemographicForm"
+import {v4 as uuidv4} from "uuid";
+import { translate } from "react-i18next";
```

Comment:
Canon comes with `react-i18next` installed out of the box, here is some documentation on how to use it: https://github.com/Datawheel/canon/tree/master/packages/core#localization